src/components/deposit_modal/deposit_modal.tsx

```diff
@@ -147,6 +147,19 @@ const DepositModal = ({
 
         globalCtx.eliminateToasts(ToastId.DEPOSIT);
         globalCtx.visibleCanceledModalHandler();
+      } else if (
+        /* Info:(20230530 - Julian) 連續入金的錯誤屬於合約內容，所以顯示沒有紅框的 Failed Modal */
+        result.code === Code.DEPOSIT_TOO_FREQUENCY
+      ) {
+        globalCtx.eliminateAllModals();
+
+        globalCtx.dataFailedModalHandler({
+          modalTitle: t('D_W_MODAL.DEPOSIT'),
+          modalContent: `${t('D_W_MODAL.FAILED_REASON_FAILED_TO_DEPOSIT')} (${result.code})`,
```

Comment:
Failed to deposit -> Deposit after 24 hrs

Reply:
Fix from backend api